spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -434,9 +446,23 @@ definitions:
             fill: STECSatElement
             desc: Array of STEC information for each space vehicle
 
- - MSG_SSR_GRIDDED_CORRECTION:
+ - MSG_SSR_GRIDDED_CORRECTION_DEP_A:
     id: 0x05F0
     short_desc: Gridded troposphere and STEC residuals
+    desc: |
+      This message was deprecated when variances (stddev)
+      were added.
+    fields:
+        - header:
+            type: GriddedCorrectionHeader
+            desc: Header of a Gridded Correction message
+        - element:
+            type: GridElement
```

Comment:
I didn't realize that the changes were in subtypes. To properly deprecate a message using subtypes you need to also change the names of the types and define a new type with the new fields.

So you'll need to deprecate the `GridElement`, `STECResidual`, and `TroposphericDelayCorrection` types by appending something like `_Dep_A` to their names and then change the names used here to use the deprecated types. Does that make sense?

Reply:
Look good now?

Reply:
Perfect! 🎉 

I'll stop making more busy work for you 😄